tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java

```diff
@@ -93,7 +93,7 @@ public void testUnusedImports() {
         BAssertUtil.validateError(result, i, "unused import module 'ballerina/jballerina.java as otherJAVA'", 2, 1);
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
Why are we disabling this?

Reply:
Need to think of a different way to write this. This uses `ballerina/io` to print a message in module init function and then capture that to a buffer and compare the outputs to verify the behaviour. With the `ballerina/io` module removed, can't do it that way. So disabled it for now, will create an issue for this.

Reply:
We can probably use interop similar to https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/execution/proj8/main.bal#L18.

Shall we create an issue if we are disabling this?

Reply:
Yeah, a util like this should work fine. Will address this separately.